### ISSUE TITLE
Enable pip cache in the travis configuration

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8420]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)| @cyli reported|
|-|-|
|Trac ID|trac#8420|
|Type|enhancement|
|Created|2016-06-04 08:23:59Z|

[#7229](https://github.com/twisted/twisted/issues/7229) added a travis configuration to run some tests using Travis-CI.

We should also enable the travis cache for pip packages.

<details><summary>Searchable metadata</summary>

```
trac-id__8420 8420
type__enhancement enhancement
reporter__cyli cyli
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1465028639249542 1465028639249542
changetime__1465066141676323 1465066141676323
version__None None
owner__Ying_Li__cyli_____ Ying Li <cyli@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_1"></a>|@cyli commented|
|-|-|

https://github.com/twisted/twisted/pull/133

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_2"></a>|@cyli set owner to @cyli<br>@cyli set status to `closed`|
|-|-|

In changeset be25eb0f148258cccf12d7e5dfb6f7293dab4cfe
```
#!CommitTicketReference repository="" revision="be25eb0f148258cccf12d7e5dfb6f7293dab4cfe"
Merge pull request #133 from alex/patch-2: Enable the travis cache

Author: alex
Reviewer: cyli
Fixes: #8420

Enable the travis cache
```
